platform/security/core/security-core-impl/src/main/java/ddf/security/assertion/impl/SecurityAssertionImpl.java

```diff
@@ -1657,6 +1517,12 @@ public void setNil(Boolean newNil) {
         public void setNil(XSBooleanValue newNil) {
 
         }
+
+        @Nonnull
```

Comment:
Fixed these by creating a new abstract class to handle all of the commonly overridden methods and these @Nonnull cases